COMOPT-1873: Launch checklist sync with ACO (#853)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/launch/index.mdx

* [ ] Update password reset links in email templates so they match your Edge Delivery Services URLs. See the [FAQ](/troubleshooting/faq#what-should-i-do-if-my-email-template-links-are-broken-after-migrating-to-edge-delivery-services-or-helix).
* [ ] Add production API keys and secrets for every integration and payment provider you use in production.
* [ ] Add your production domains to allowlists where your backends require it, and send a test webhook to confirm delivery.
* [ ] Restrict Cross-Origin Resource Sharing (CORS) to the origins your storefront and tools actually use. See [CORS setup](/setup/configuration/cors-setup/).
* [ ] Publish a privacy policy and a cookie consent flow that match your regions, including General Data Protection Regulation (GDPR) or California Consumer Privacy Act (CCPA) rules when they apply.
</Checklist>

<Checklist checklistKey="cdn">
### CDN and caching

> * [ ] Point your CDN, Sidekick extensions, sitemap jobs, and image importer to the same production GraphQL endpoints your storefront uses in `config.json`. For Adobe Commerce on Cloud or on-premises (PaaS), this is typically `commerce-core-endpoint`, for example, `https://yourstore.example.com/graphql`, and sometimes a separate `commerce-endpoint` when Catalog Service handles catalog reads. For Adobe Commerce as a Cloud Service (ACCS) or Adobe Commerce Optimizer (ACO), use `commerce-endpoint` on the Adobe Commerce API host, for example, `https://na1-sandbox.api.commerce.adobe.com/your-environment-id/graphql`. Replace the region, tier, and path ID with your production values from Adobe (see [Commerce configuration](/setup/configuration/commerce-configuration/)). For ACO, also configure `commerce-core-endpoint` when checkout or account services still rely on a separate core Commerce host.

Reviewer: ```suggestion
* [ ] Point your CDN, Sidekick extensions, sitemap jobs, and image importer to the same production GraphQL endpoints your storefront uses in `config.json`.
  * **For Adobe Commerce on Cloud or on-premises (PaaS)**, this is typically `commerce-core-endpoint`, for example, `https://yourstore.example.com/graphql`, and sometimes a separate `commerce-endpoint` when Catalog Service handles catalog reads.
  * **For Adobe Commerce as a Cloud Service** or Adobe Commerce Optimizer**, use `commerce-endpoint` on the Adobe Commerce API host, for example, `https://na1-production.api.commerce.adobe.com/your-environment-id/graphql`.  See [Commerce configuration](/setup/configuration/commerce-configuration/)).
  * **For Adobe Commerce Optimizer**, also configure `commerce-core-endpoint` when checkout or account services still rely on a separate core Commerce host.
```

* [ ] Test your [CDN configuration](/setup/configuration/content-delivery-network/) and confirm cache hits, manual purges, and automatic invalidation behave as you expect.
* [ ] Publish a small content change and confirm it appears on the production domain without stale HTML (push invalidation works end to end).
* [ ] For [multi-store setups](/setup/seo/indexing/#multi-store-setups), add a store-specific cache buster to Catalog Service and Live Search requests (for example, a query string or a CDN rule) so each storefront cache stays separate.
* [ ] Lower DNS time-to-live (TTL) values a few days before cutover so DNS changes propagate quickly.
* [ ] Confirm that DNS A and CNAME records for every hostname point to the right targets.
* [ ] Confirm that the SSL/TLS certificate is provisioned and verified for the production domain and that HTTPS is enforced everywhere.
* [ ] Confirm that visitors who type the apex domain and visitors who type `www` both land on the URL you want.
</Checklist>

<Checklist checklistKey="catalog">
### Catalog service

> * [ ] Switch to the Catalog Service production endpoint (`https://catalog-service.adobe.io/graphql`). For how Adobe Commerce exposes and secures the service, see the <Link href={"https://experienceleague.adobe.com/en/docs/commerce/catalog-service/installation#access-the-service"} text={"Catalog Service installation guide"} /> on Experience League.

Reviewer: ```suggestion
* [ ] In your storefront configuration, switch to the production endpoint for Catalog or Merchandising Services.

  * **For Adobe Commerce on Cloud or on-premises (PaaS)**, switch the `commerce-endpoint` to `https://catalog-service.adobe.io/graphql`. If the Adobe Commerce Optimizer Connector is configured for your project, use the Adobe Commerce Optimizer GraphQL endpoint.
  * **For Adobe Commerce as a Cloud Service** or **Adobe Commerce Optimizer**,  switch the `commerce-endpoint` to the Commerce GraphQL production endpoint for your instance, for example `https://na1-production.api.commerce.adobe.com/your-environment-id/graphql`.  See [Commerce configuration](/setup/configuration/commerce-configuration/)).
```

> * [ ] Ensure a production environment is configured in the Commerce Services Connector (will result in a new `environmentId`).

Reviewer: ```suggestion
* [ ] For Commerce on cloud or on-premises only, ensure a production environment is configured in the Commerce Services Connector (will result in a new `environmentId`). See [Select or create a SaaS project](https://experienceleague.adobe.com/en/docs/commerce/user-guides/integration-services/saas#createsaasenv).
```

* [ ] Record the P1 support hotline number: US (+1) 800-497-0335, then press 3 for the Adobe Commerce P1 hotline.
* [ ] Train your team to open a support ticket before calling the P1 hotline.
* [ ] Check for the latest boilerplate changes and update your project accordingly. See [Monitor boilerplate changes](/boilerplate/updates/#monitor-boilerplate-changes).
* [ ] Confirm that a disaster recovery plan exists and has been tested.
* [ ] After launch, verify Lighthouse scores on the production domain.
* [ ] After launch, monitor Google Search Console for indexing and crawl errors.
* [ ] After launch, monitor 404 reports and add redirects for high-traffic legacy URLs.
* [ ] After launch, confirm that MarTech and analytics data appears on production.
* [ ] Ask your Adobe Customer Technical Advisor (CTA), Customer Service Engineer (CSE), or Account Manager (AM) to enable high-SLA monitoring.
* [ ] Establish a process to track and upgrade boilerplate and extension packages to current versions. Start from [Updates](/boilerplate/updates/) and [Release Information](/releases/).
</Checklist>

## Documentation

> Edge Delivery Services launch topics (DNS, CDN vendors, redirects, and platform checks) live on the Adobe Experience Manager documentation site. Start from the <Link href={"https://www.aem.live/docs/#launch"} text={"Launch hub for Edge Delivery Services"} />, or use the table below. The checklist sections above focus on Adobe Commerce Storefront tasks on this site.

Reviewer: ```suggestion
Edge Delivery Services launch topics (DNS, CDN vendors, redirects, and platform checks) are available on the Adobe Experience Manager documentation site. Start from the <Link href={"https://www.aem.live/docs/#launch"} text={"Launch hub for Edge Delivery Services"} />, or use the table below. The checklist sections above focuses on Adobe Commerce Storefront tasks for sites integrated with an Adobe Commerce backend.
```

</Checklist>

<Checklist checklistKey="prerendering">
### Enable pre-rendering

* [ ] Turn on pre-rendering for the pages that matter most at launch. See [Pre-rendering](/setup/configuration/aem-prerender/).
* [ ] Use lowercase URLs everywhere so pre-rendered links stay valid.
* [ ] View page source on a pre-rendered page and confirm you see metadata and main body content in the first HTML response.
* [ ] For each language or region you support, open a sample page and confirm the translated content loads.
* [ ] Add any extra HTML your project needs (for example, analytics snippets that must appear in the initial HTML).
</Checklist>

<Checklist checklistKey="performance">
### Performance and monitoring

* [ ] Follow [Performance best practices](/get-started/performance/) for your storefront.
* [ ] (Optional) Set up Google Analytics and Google Tag Manager.
* [ ] Validate your <Link href={"https://github.com/adobe/commerce-events/tree/main/examples/events/snowplow-debugger"} text={"storefront\n      events"} /> implementation, then confirm events appear in Live Search and Product Recommendations dashboards in the Adobe Commerce Admin.

> * [ ] Validate that the `environment` field in your [Commerce configuration](/setup/configuration/commerce-configuration) is set to `"Testing"` while you build, then change it to `"Production"` when you launch. See [Analytics](/setup/analytics/instrumentation/).

Reviewer: Can you link directly to the analytics section --it's a pretty long page.  Not sure of format for bookmark links, is it:

/setup/configuration/commerce-configuration/#analytics

* [ ] Confirm that your Lighthouse scores are green on key templates and aim for a strong score (for example, `100` on mobile and desktop where practical), using the same checks you already ran on your `.aem.live` preview site.
</Checklist>

<Checklist checklistKey="analytics">

> ### Analytics and monitoring

Reviewer: Check the following Analytics and monitoring section and add useful links to related documentation where relevant (ask experts reviewers)

* [ ] Add Real User Monitoring (RUM) before launch so you can compare performance before and after you switch traffic.
* [ ] If you use Adobe Experience Platform, confirm data collection is configured for the production site.
* [ ] On the production hostname, confirm that marketing technology (MarTech) tags fire and send data to the right tools.
* [ ] Write down current analytics baselines (page views, bounce rate, and similar metrics) so your team expects a shift after the new site ships.
* [ ] Walk a test order from add to cart through checkout to the order confirmation page and confirm each step is tracked.
</Checklist>

<Checklist checklistKey="security">
### Security and access

* [ ] Set permissions for Document Authoring (DA.live) content and Edge Delivery Services sites so only trusted people can publish. See <Link href={"https://da.live/docs/administration/permissions"} text={"DA.live permissions"} /> and <Link href={"https://www.aem.live/docs/authentication-setup-authoring"} text={"Edge Delivery Services authentication setup for authoring"} />.

> * [ ] Confirm that the product visuals integration is turned on for your project. See <Link href={"https://experienceleague.adobe.com/en/docs/experience-manager-learn/cloud-service/accessing/overview#"} text={"Experience Manager Cloud Service access overview for product visuals"} />.

Reviewer: Comment applies to line #98 

This link doesn't not provide immediately obvious information.  - Consider:    Confirm that the Product Visuals integration is available for your project. For details on the required entitlements, prerequisites, and configuration, see the [AEM Assets Integration for Commerce Guide](https://experienceleague.adobe.com/en/docs/commerce/aem-assets-integration/overview#next-steps)

* [ ] Confirm that coupons, discounts, and loyalty programs behave as expected.
* [ ] Complete user acceptance testing (UAT) on staging and production.
* [ ] Complete load and stress testing and share results with your Adobe team.
* [ ] Confirm that page load time is under three seconds on desktop and mobile.
* [ ] Confirm that images, scripts, and assets are optimized.
* [ ] Test Chrome, Firefox, Safari, and Edge for consistent behavior.
* [ ] Confirm that responsive layouts work on mobile, tablet, and desktop.
* [ ] Test performance on 3G, 4G, and Wi-Fi connections.
* [ ] Complete an accessibility audit (WCAG, screen reader, keyboard navigation).
* [ ] Establish a post-launch 404 monitoring plan.
* [ ] Test your rollback plan to confirm it works if launch issues occur.
</Checklist>

<Checklist checklistKey="management">
### Launch day and post-launch

* [ ] Confirm your launch date with Adobe and notify your CTA, CSE, or AM so they can coordinate support during the go-live.
* [ ] Record the P1 support hotline number: US (+1) 800-497-0335, then press 3 for the Adobe Commerce P1 hotline.
* [ ] Train your team to open a support ticket before calling the P1 hotline.

> * [ ] Check for the latest boilerplate changes and update your project accordingly.

Reviewer: ```suggestion
* [ ] Check for the [latest boilerplate changes](https://experienceleague.adobe.com/developer/commerce/storefront/boilerplate/updates/#monitor-boilerplate-changes) and update your project accordingly.
```

* [ ] Confirm that a disaster recovery plan exists and has been tested.
* [ ] After launch, verify Lighthouse scores on the production domain.
* [ ] After launch, monitor Google Search Console for indexing and crawl errors.
* [ ] After launch, monitor 404 reports and add redirects for high-traffic legacy URLs.
* [ ] After launch, confirm that MarTech and analytics data appears on production.

> * [ ] Ask your Adobe CTA, CSE, or AM to enable high-SLA monitoring.

Reviewer: ```suggestion
* [ ] Ask your Adobe Customer Technical Advisor (CTA), Customer Service Engineer (CSE), or Account Manager (AM) to enable high-SLA monitoring.
```
